Makefile

```diff
@@ -21,7 +21,11 @@ test:
 
 runmigrations:
 	env/bin/python wellregistry/manage.py migrate --database=postgres postgres
-	env/bin/python wellregistry/manage.py migrate registry 0000
+	env/bin/python wellregistry/manage.py migrate admin
+	env/bin/python wellregistry/manage.py migrate auth
+	env/bin/python wellregistry/manage.py migrate contenttypes
+	env/bin/python wellregistry/manage.py migrate sessions
+	env/bin/python wellregistry/manage.py migrate social_django
 	env/bin/python wellregistry/manage.py migrate registry
```

Comment:
unless the order is important, I think we can migrate in one command.
`env/bin/python wellregistry/manage.py migrate`

Reply:
Good point. I was having issues earlier where it was helpful to have the order defined. I will test this out.